assets/js/script.js

```diff
@@ -1,13 +1,19 @@
-$(window).ready(function () {
+window.ready = async function () {
 
-    let getCurrency = $.getJSON('https://api.exchangeratesapi.io/latest');
-    let rub = getCurrency.responseJSON.rates.RUB;
+
+    let getApiCurrency = await fetch('https://api.exchangeratesapi.io/latest?symbols=USD,RUB');
+
+    console.log(getApiCurrency.json());
+
+
+    await (getCurrency);
+    // let rub = getCurrency.responseJSON.rates.RUB;
     setInterval(startParse, 1000);
     setTimeout(startParse, 0);
 
     function startParse() {
-        let x = document.querySelectorAll('.price:not(.checked-item), div.product-price:not(.checked-item), #price_inside_buybox:not(.checked-item), #priceblock_dealprice:not(.checked-item), .priceBlockStrikePriceString:not(.checked-item), .goodsIntro_price:not(.checked-item), .gbGoodsItem_price:not(.checked-item)');
```

Comment:
need to store seperatly for each site not in 1 variable, its a MESS
Anyone suggest to me the best method to store classes.
Array do not suggest :)